pyaedt/modeler/PrimitivesNexxim.py

```diff
@@ -346,7 +346,7 @@ def create_field_model(self, design_name, solution_name, pin_names, model_type="
         return False
 
     @aedt_exception_handler
-    def create_resistor(self, compname=None, value=50, xpos=0, ypos=0, angle=0, use_instance_id_netlist=False):
+    def create_resistor(self, compname=None, value=50, location=None, angle=0, use_instance_id_netlist=False):
```

Comment:
```suggestion
    def create_resistor(self, compname=None, value=50, location=[], angle=0, use_instance_id_netlist=False):
```
If location is a list, it would be better so set to an empty list. It still returns ``False`` when checked.